Doc/c-api/init_config.rst

```diff
@@ -828,6 +828,18 @@ PyConfig
 
       Default: ``0``.
 
+   .. c:member:: int32_t int_max_str_digits
+
+      If greater than 0, enable int conversion digit limitations. ``-1`` means
+      that :data:`sys.int_info.default_max_str_digits` will be used.
+
+      Configured by the :option:`-X int_max_str_digits <-X>` command line
+      flag or the :envvar:`PYTHONINTMAXSTRDIGITS` environment varable.
+
```

Comment:
You might document that there is a (constant) minimum value: _PY_LONG_MAX_STR_DIGITS_THRESHOLD.